src/views/DatasetManager/components/CreateDatasetButton/components/CreateDatasetWizard/CreateDatasetWizard.js

```diff
@@ -297,7 +161,9 @@ export const CreateDatasetWizard = ({ open, closeDialog }) => {
               </Stepper>
             </Grid>
             {activeStep === 0 && firstStep}
-            {activeStep === 1 && secondStep}
+            {activeStep === 1 && (
+              <DatasetCreationParameters dataSourceType={dataSourceType} setDataSourceType={setDataSourceType} />
```

Comment:
No, it can't be. We definitively need to declare this useState inside the wizard in order to avoid the useState resetting when the user goes back to the first step (to avoid the sourceType selected by the user being lost when back to the first step).